libs/logger.js

```diff
@@ -0,0 +1,22 @@
+import winston from 'winston';
+
+/**
+ * Log function that outputs console logs in json format.
+ *
+ * @param {('error'|'warn'|'info'|'http'|'verbose'|'debug'|'silly')} level Log level based on npm log level standard.
+ * @param {string} message   Log message.
+ * @param {string} requestId Request id to be able to filter logs on a single request.
+ * @param {string} errorCode Error code for locating log in code.
+ * @param {object} data      Custom data related to log.
+ * @returns {void}
+ */
+export const logger = (level, message, requestId, errorCode, data = {}) => {
```

Comment:
Calling it `logger` seems off to me, as nouns are usually reserved for classes etc. I would suggest renaming it to a verb like `log` (but maybe that's too short/conflicting) 🙂.

Reply:
will do